src/Canvas.cc

```diff
@@ -234,31 +234,32 @@ Canvas::EIO_AfterToBuffer(eio_req *req) {
  */
 
 NAN_METHOD(Canvas::ToBuffer) {
-  NanScope();
+  Nan::HandleScope scope;
   cairo_status_t status;
   uint32_t compression_level = 6;
   uint32_t filter = PNG_ALL_FILTERS;
-  Canvas *canvas = ObjectWrap::Unwrap<Canvas>(args.This());
+  Canvas *canvas = Nan::ObjectWrap::Unwrap<Canvas>(info.This());
 
   // TODO: async / move this out
   if (canvas->isPDF() || canvas->isSVG()) {
     cairo_surface_finish(canvas->surface());
     closure_t *closure = (closure_t *) canvas->closure();
 
-    Local<Object> buf = NanNewBufferHandle((char*) closure->data, closure->len);
-    NanReturnValue(buf);
+    Local<Object> buf = Nan::NewBuffer((char*) closure->data, closure->len).ToLocalChecked();
+    info.GetReturnValue().Set(buf);
+    return;
   }
 
-  if (args.Length() > 1 && !(args[1]->StrictEquals(NanUndefined()) && args[2]->StrictEquals(NanUndefined()))) {
-    if (!args[1]->StrictEquals(NanUndefined())) {
+  if (info.Length() > 1 && !(info[1]->StrictEquals(Nan::Undefined()) && info[2]->StrictEquals(Nan::Undefined()))) {
```

Comment:
I think we can just use `IsUndefined()` for these instead of the whole `StrictEquals` dance